public/app/core/utils/file_export.ts

```diff
@@ -75,12 +84,22 @@ export function convertSeriesListToCsv(seriesList, dateTimeFormat = DEFAULT_DATE
   return text;
 }
 
-export function exportSeriesListToCsv(seriesList, dateTimeFormat = DEFAULT_DATETIME_FORMAT, excel = false) {
-  const text = convertSeriesListToCsv(seriesList, dateTimeFormat, excel);
+export function exportSeriesListToCsv(
+  seriesList,
+  dateTimeFormat = DEFAULT_DATETIME_FORMAT,
+  excel = false,
+  timeZone?: TimeZone
+) {
+  const text = convertSeriesListToCsv(seriesList, dateTimeFormat, excel, timeZone);
   saveSaveBlob(text, EXPORT_FILENAME);
 }
 
-export function convertSeriesListToCsvColumns(seriesList, dateTimeFormat = DEFAULT_DATETIME_FORMAT, excel = false) {
+export function convertSeriesListToCsvColumns(
```

Comment:
IMO up to 3 args is fine, better just 2, with the second being an options object, with a proper type definition.